plugins/org.eclipse.xtext.xbase/src/org/eclipse/xtext/xbase/compiler/JvmModelGenerator.xtend

```diff
@@ -122,9 +122,13 @@ class JvmModelGenerator implements IGenerator {
 		if(type.qualifiedName != null)
 			fsa.generateFile(type.qualifiedName.replace('.', '/') + '.java', type.generateType(generatorConfigProvider.get(type)))
 	}
+
+	protected def organizeImports() {
+		true
+	}
 	
 	def CharSequence generateType(JvmDeclaredType type, GeneratorConfig config) {
-		val importManager = new ImportManager(true, type)
+		val importManager = new ImportManager(organizeImports, type)
```

Comment:
I think a method `createImportManager(JvmDeclaredType)` would be even better since clients could easily provide a custom instance if there is a bug in the default impl.